### PR TITLE
unit test cases

### PR DESCRIPTION
**Pull request by MashB**

### CODE REVIEW

File: src/__test__/health.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            url: "http://localhost:8080",
+            query: {}
+        } as Request;
+        let responseObj = {};
+        const mockResponse: Partial<Response> = {
+            send: jest.fn().mockImplementation((result) => {
+                responseObj = result;
+            })
+        };
+        const users = await healthController.getping(mockRequest, mockResponse as Response);
```
Comment (susrisha) [inline src/__test__/health.test.ts]: `users` ??

File: src/__test__/pathways.test.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                responseObj = result;
+            })
+        };
+
+        const list: GtfsPathwaysDTO[] = [new GtfsPathwaysDTO()]
+        const spy = jest
+            .spyOn(gtfsPathwaysService, "getAllGtfsPathway")
+            .mockResolvedValueOnce(list);
+
+        const users = await gtfsPathwaysController.getAllGtfsPathway(mockRequest, mockResponse as Response);
```
Comment (susrisha) [inline src/__test__/pathways.test.ts]: change this constant `users`

**Review by susrisha — CHANGES_REQUESTED** [review]
Minor changes requested

File: src/__test__/health.test.ts (hunk 1 above)
Comment (MashB) [reply src/__test__/health.test.ts]: done

File: src/__test__/pathways.test.ts (hunk 2 above)
Comment (MashB) [reply src/__test__/pathways.test.ts]: done

**Review by susrisha — APPROVED** [review]
Approved

**Pull request merged**